### DISCUSSION TITLE
Prevent loading of higher levels

### DISCUSSION THREAD

**Discussion by AndiLeni** [openlayers/openlayers#15900] [Q&A]
Hello,

I have a map consiting of TileLayer(TileWMS) and TileLayer(VectorTileLayer). 
Is it somehow possible to restrict loading high zoom levels to reduce bandwith?

The XYZ source has a maxZoom option  which works exactly like I'd expect: it is visible on lower levels, but does not load new data.
The maxZoom option for TileWMS and VectorTileLayer however does something different: it hides the layer completely below the set level.

Is there an option to correct this behavior?

Thanks and kind regards

**Comment by ahocevar** [comment]
To achieve what you want, you have to set the maxZoom on the source, i.e. the VectorTile source, not the VectorTile layer. For the TileWMS source, you'll have to provide a `tileGrid` to the constructor, e.g. `createXYZ({maxZoom: 10})` - TileWMS does not have a `maxZoom` option.
